src/test/e2e/variable_test.go

```diff
@@ -153,12 +153,12 @@ func TestBundleWithEnvVarHelmOverrides(t *testing.T) {
 	deploy(t, bundlePath)
 
 	// check override variables, ensure they are coming from env vars and take highest precedence
-	cmd := strings.Split("zarf tools kubectl get deploy -n podinfo unicorn-podinfo -o=jsonpath='{.spec.template.spec.containers[0].env[?(@.name==\"PODINFO_UI_COLOR\")].value}'", " ")
+	cmd := strings.Split("z tools kubectl get deploy -n podinfo unicorn-podinfo -o=jsonpath='{.spec.template.spec.containers[0].env[?(@.name==\"PODINFO_UI_COLOR\")].value}'", " ")
```

Comment:
Might make sense to leave one of these as `zarf` rather than `z` so that both are tested? (And ignore if that's already happening in a separate test, just staring at the diffs so might be missing somewhere else)

Reply:
Jk, scrolled up a few lines and there's others with the full command still.

Reply:
yeah, the other test files (bundle, runner, main) use "zarf" as well